src/python/bot/fuzzers/syzkaller/engine.py

```diff
@@ -93,6 +99,27 @@ def _create_temp_corpus_dir(self, name):
     engine_common.recreate_directory(new_corpus_directory)
     return new_corpus_directory
 
+  def get_phone_family(self):
+    fingerprint = settings.get_build_fingerprint()
+    matcher = re.search(FINGERPRINT_REGEX, fingerprint)
+    return matcher.group(1)
+
+  def save_corpus(self, source, destination):
+    source_file = os.path.join(source, CORPUS_FILENAME)
+    if not os.path.exists(destination):
```

Comment:
for these two lines, we already have a helper in
from system import shell
shell.create_directory(destination)
it auto does the directory creation check. same for init_corpus.

Reply:
Done.